Autopilot/PathManager/Inc/pathModeSelector.hpp

```diff
@@ -34,6 +39,21 @@ class PathModeSelector {
          * @return none
          */
         void execute(Telemetry_PIGO_t telemetry_in, SFOutput_t sensor_fusion_in, IMU_Data_t imu_data_in);
+        
+        /**
+         * Checks whether the instruction queue is empty.
+         * 
+         * @return true if the queue is empty. Else return false
+         */
+        bool instructionQueueIsEmpty();
```

Comment:
To make this more specific to cruising mode, maybe name the function something like:

`cruisingEditPathInstrQueueIsEmpty()` or something else if you have ideas

that way if we ever need to add new queues we can easily tell which queue we are calling.